content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,160 @@
+      6. Read Models: Data required to make decisions or perform commands
+      Note: it does not always need to happen in this order and more grammar may be added, so it is best to leave enough space or be prepared to shift post-its further to the right as you make changes.
+    - Negative/Alternative Outcomes: If a command can result in multiple or alternative events then all events should be placed vertically
+    - Pain Points and Opportunities: Place a sticky to represent any pain points and any opportunities that are identified at the time. Pain points can be around the process itself or the current system or tools supporting the process
+
+3. Narrate the process again, including the policies and commands and add or correct any grammar along the way. 
+   You can repeat this until you have agreed on at least one end-to-end slice through the process
+
+4. Review Questions and Pain Points: Try to answer as many questions as possible and rationalise pain points and opportunities
+
+#### Success/Expected Outcomes
+   - Capture information to assist further in the development process
+     - Ubiquitous Language
+     - Domain Events
+     - Read Models
+     - Manual Actions + System Commands
+     - Decision Making Policies
+     - External Integration
+     - Existing Applications
+     - Pain Points + Opportunities
+   - Identify one or more thin slices. Pick out a set of events that represents an end-to-end path through the process. Usually you will start with a simplified version of the `golden path` and add more slices as you choose to tackle more complexity and more outcomes. 
+
+## Related Practices:
+### EventStorming
+As mentioned Process Modelling evolved from EventStorming and is a fundamental part of the complete application of Event Storming as prescribed by Alberto Brandolini. However the storming phase, or `Big Picture` as referred to by Alberto Brandolini, has been more widely adopted with minor variations to the original, such as the use of [Event Storming](/practices/event-storming)  as described in this Developer Center. Therefore Process Modelling naturally offers a way to enrich any EventStorming session
+### UML/C4/Boris
+if you are using an Object Oriented based notation to model your architecture, Process Modelling offers an algorithmic view of the problem you are solving, and can be used to complete a number of diagrams as the process is narrated.
+### Boris and SNAP
+The grammar used by Process Modelling offers a lot of information that can be directly used in the completion of [Boris](/practices/boris) and [SNAP](/practices/boris) exercises. 
+
+### Variations:
+### Event Storming | Software Modelling
+This is an extension of Process Modelling and is represented by Alberto Brandolini as a third phase in the EventStorming methodology. It is primarily used to identify an aggregate responsible for the execution of one or more commands.
+#### Sample Agenda & Prompts
```

Comment:
This is interesting; we haven't done this before. Here you have submitted an entire alternative practices/method within the "Variations" section. 

I suggest extracting this section as it's own practice article submission If the variation is significant enough to warrant an entire re-stating of the entire session.

Reply:
They are actually quite similar. I have changed this to focus more on the variation rather than just restating the whole process

Reply:
Perhaps you could have a look and let me know what you think of my changes?